cwltest/utils.py

```diff
@@ -73,6 +76,21 @@ def compare_location(expected, actual):
         return
     if actual.get("class") == "Directory":
         actual[comp] = actual[comp].rstrip("/")
+        exist_fun = os.path.isdir  # type: Callable[[str], bool]
```

Comment:
Can you add a test case involving a Directory?

Reply:
Added!